dashboard/app/controllers/projects_lists_controller.rb

```diff
@@ -0,0 +1,41 @@
+class ProjectsListsController < ApplicationController
```

Comment:
Good question, this should probably really go in the projects controller, especially since the students view of their own projects will replace what you see today at `/projects/`, which is currently served from the `ProjectsController`. I'll add this to the `ProjectsController` as `/projects/section/:id` or something like that.